src/Kunstmaan/SeoBundle/Twig/SeoTwigExtension.php

```diff
@@ -244,11 +244,16 @@ public function setWebsiteTitle($websiteTitle)
     /**
      * @param $src
      *
-     * @return Seo
+     * @return array|null
      */
     public function getImageDimensions($src)
     {
-        list($width, $height) = getimagesize($src);
+        try {
+            list($width, $height) = getimagesize($src);
+        } catch (\Exception $e) {
+            return null;
+        }
+
```

Comment:
bare in mind that the file can be stored on a remote server. I turned this feature altogether since my media files are stored in a S3 bucket